src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2640,7 +2640,7 @@ class S4Importer final : public IParkImporter
     void ClearExtraTileEntries()
     {
         // Reset the map tile pointers
-        std::fill(std::begin(gTileElementTilePointers), std::end(gTileElementTilePointers), nullptr);
+        std::memset(gTileElementTilePointers, 0, sizeof(gTileElementTilePointers));
```

Comment:
There's a reason why you can't `std::fill` anymore. With `gTileElementTilePointers[][]` the first-order array is of type `gTileElementTilePointers*[]` so here you're effectively trying to set each of those pointers to `nullptr`.